src/components/BeamContext.tsx

```diff
@@ -87,18 +88,20 @@ export function BeamProvider({ children, ...presentationProps }: BeamProviderPro
   return (
     <BeamContext.Provider value={{ ...context }}>
       <PresentationProvider {...presentationProps}>
-        <AutoSaveStatusProvider>
-          <SnackbarProvider>
-            {/* OverlayProvider is required for Modals generated via React-Aria */}
-            <ToastProvider>
-              <OverlayProvider>
-                {children}
-                {modalRef.current && <Modal {...modalRef.current} />}
-              </OverlayProvider>
-              <SuperDrawer />
-            </ToastProvider>
-          </SnackbarProvider>
-        </AutoSaveStatusProvider>
+        <RightPaneProvider>
```

Comment:
@stephenh @bdow - relocated `RightPaneProvider` to the beam context, want to make sure you both don't have any concerns with this living at this level. - It feels like the most ideal location for it when testing in blueprint.

Reply:
Yep, I think it's the right place for it.